mypyc/rt_subtype.py

```diff
@@ -51,5 +51,12 @@ def visit_rtuple(self, left: RTuple) -> bool:
                 is_runtime_subtype(t1, t2) for t1, t2 in zip(left.types, self.right.types))
         return False
 
+    def visit_rstruct(self, left: RStruct) -> bool:
+        if isinstance(self.right, RStruct):
+            # TODO: should we consider names?
```

Comment:
If we use this to allow assignments from one struct to another in generated C (I'm not sure but I think we might), we should consider names of both the struct and the fields, since C structs don't support structural subtyping.